src/styles/utils/variables.scss

```
@@ -0,0 +1,15 @@
+$clock-size: 300px;
+$circle: 50%;
+$minutes-arrow-height: 50%;
+$minutes-arrow-width: 2%;
+$minutes-arrow-color: green;
```

This is more readable, when you read css without looking into variables.scss file
```suggestion
$color-green: green;
```